cmd/kagome/tokenize/cmd.go

```diff
@@ -75,6 +77,9 @@ func (o *option) parse(args []string) (err error) {
 	if o.mode != "" && o.mode != "normal" && o.mode != "search" && o.mode != "extended" {
 		return fmt.Errorf("invalid argument: -mode %v\n", o.mode)
 	}
+	if o.sysdic != "" && o.sysdic != "normal" && o.sysdic != "simple" {
```

Comment:
__[gometalinter]__
* [goconst] 2 other occurrence(s) of "normal" found in: cmd.go:77:31 cmd.go:134:7

<!-- Comment by [SideCI](https://sideci.com) -->